aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1316,7 +1316,7 @@ at::Tensor PackedConvWeightsOnednn<kSpatialDim>::apply_impl(
     op_attr = kReluFused ? ideep::attr_t::fuse_relu() : ideep::attr_t();
   }
   // Since src zero point is unknown, set runtime value here
-  op_attr.set_zero_points(DNNL_ARG_SRC, ideep::utils::tensor_zp_mask(1), {DNNL_RUNTIME_S32_VAL});
+  op_attr.set_zero_points(DNNL_ARG_SRC, ideep::utils::tensor_zp_mask(1), src_zero_points);
```

Comment:
Are we use src zero point to initialize the primitive in the cache or use `DNNL_RUNTIME_S32_VAL` instead? If it is the latter, the primitive cache won't take effect any longer?

Reply:
> Are we use src zero point to initialize the primitive in the cache or use `DNNL_RUNTIME_S32_VAL` instead? If it is the latter, the primitive cache won't take effect any longer?

We used to use runtime value to create primitive and put it in cache. This PR uses true values of src zero point to create a primitive.
Src zero point is part of the cache key. If src zero point changes, cache misses. It has always been this behavior, not introduced by this PR.

Reply:
OK. If we were using `DNNL_RUNTIME_S32_VAL` previously, the src zero point shouldn't be part of the key? Anyway, the change looks good then.